tests/spread/core22/components/task.yaml

```diff
@@ -11,13 +11,19 @@ restore: |
 execute: |
   snapcraft pack
 
-  # assert contents of default partition
+  # assert snap contents
   unsquashfs -dest "snap-contents" "hello-components_1.0_amd64.snap"
   if [ ! -e "snap-contents/bin/hello" ]; then
     echo "Expected 'bin/hello' in snap contents"
     exit 1
   fi
 
+  # assert snap metadata
+  if ! diff snap-contents/meta/snap.yaml expected-snap.yaml; then
```

Comment:
we are doing diff -u in other places (grep the code), might be good to stay consistent here

Reply:
Added.